.pylintrc

```diff
@@ -1,3 +1,5 @@
+
```

Comment:
Why the blank lines in this and next file?

Reply:
I tried to get them out.
But this is something coming from the main template repo:
https://github.com/OCA/edi/pull/686/files#diff-63a9c44a44acf85fea213a857769990937107cf072831e1a26808cfde9d096b9